### PR TITLE
[LIT-178] Create version 4.3.0

### PR DESCRIPTION
**Pull request by Heckfer**
### Changes

- No more message-contracts (from previous 4.2.x versions)
- Updated `Formatter`
- Updated `Credo`
- `Poison` replaced by `Jason`

### CODE REVIEW

File: test/kafka_message_bus/producer_test.exs [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
-
-      [produced_message] = TestAdapter.get_produced_messages()
-
+      assert [produced_message] = TestAdapter.get_produced_messages()
       assert produced_message["data"] == message
     end
 
-    test "it fails to produce to topics that have no adapters" do
-      message = %{"data" => "here"}
+    test "produces messages without not loaded associations on it" do
```
Comment (execb5) [inline test/kafka_message_bus/producer_test.exs:19]:
```suggestion
    test "produces messages without unloaded associations on it" do
```

File: .credo.exs [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        #
+        # You can give explicit globs or simply directories.
+        # In the latter case `**/*.{ex,exs}` will be used.
+        #
+        included: [
+          "lib/",
+          "src/",
+          "test/",
+          "web/",
+          "apps/*/lib/",
```
Comment (goalves) [inline .credo.exs:29]: You can remove `src` and `web` and `apps` since those folders don't exist in this project

File: .credo.exs [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+          "apps/*/src/",
+          "apps/*/test/",
+          "apps/*/web/"
+        ],
+        excluded: [~r"/_build/", ~r"/deps/", ~r"/node_modules/"]
+      },
+      #
+      # Load and configure plugins here:
+      #
```
Comment (goalves) [inline .credo.exs:38]: remove node_modules from the list of regexes

File: .formatter.exs [hunk 4]
```diff
@@ -1,20 +1,5 @@
...
-    assert_same: 2,
-
-    # Errors tests
-    assert_eval_raise: 3,
-
-    # Mix tests
-    in_fixture: 2,
-    in_tmp: 2
-  ]
+  inputs: ["*.{ex,exs}", "priv/*/seeds.exs", "{config,lib,test}/**/*.{ex,exs}"],
```
Comment (goalves) [inline .formatter.exs:2]: Thsi library does not have a priv* folder

File: .formatter.exs [hunk 5]
```diff
@@ -1,20 +1,5 @@
...
+  line_length: 120,
+  subdirectories: ["priv/*/migrations"]
```
Comment (goalves) [inline .formatter.exs:4]: Same thing here :)

File: test/kafka_message_bus/producer_test.exs (hunk 1 above)
Comment (Heckfer) [reply test/kafka_message_bus/producer_test.exs:19]: i believe `not loaded` is better since that's now the changeset calls it

**Review by amarantedaniel — APPROVED** [review]

**Review by goalves — APPROVED** [review]

**Comment by edaroit** [comment]
Closed due to the creation of #33, which is the same PR but with the correct branch.

**Pull request closed without merging**